corehq/project_limits/models.py

```diff
@@ -80,3 +81,12 @@ class RateLimitedTwoFactorLog(models.Model):
     window = models.CharField(max_length=15, null=False)
     # largest input is 'number_rate_limited', 31 for headroom
     status = models.CharField(max_length=31, null=False)
+
+
+class FeatureLimit(models.Model):
```

Comment:
I thought these would have a domain, too - I added @nospame as a reviewer because this sounds like the spec he just wrote. Am I getting two features confused? @gherceg can you give an example of a feature?

Reply:
Thanks for highlighting this Jenny! I was not aware of this and Evan and I offlined about it. It sounds like we are implementing two different things, where Evan's model is defining usage limits for specific projects/subscription, and this one is intended for system wide limits. So I'll likely rename this model to `SystemLimit` to align with that difference.

As for expecting these to have a domain, that is interesting. The standard case should not require a domain field, but in the event of an emergency, it would be nice to define a limit for a specific domain (whether higher or lower). So I'm tempted to add a domain field that defaults to null (meaning it applies to all domains), and we should only consider adding a specific domain entry in extreme circumstances. What do you think?

Reply:
I'd lean towards keeping it simpler for now. I can envision the domain-specific limits you're describing, but let's walk before we start running.